<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>银行管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/Icon/Icon/pay-blue-48.png</normaloff>:/Icon/Icon/pay-blue-48.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonDeposit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
    </property>
    <property name="text">
     <string>存款</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Icon/Icon/deposit-white-48.png</normaloff>:/Icon/Icon/deposit-white-48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonWithdraw">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
    </property>
    <property name="text">
     <string>取款</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Icon/Icon/withdraw-white-48.png</normaloff>:/Icon/Icon/withdraw-white-48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Icon/Icon/config-white-48.png</normaloff>:/Icon/Icon/config-white-48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>卡号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>银行：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>身份证号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>联系方式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>卡内金额：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelNum">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBank">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelID">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelContact">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBalance">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
    </property>
    <property name="text">
     <string>安全退出</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/Icon/Icon/exit-white-48.png</normaloff>:/Icon/Icon/exit-white-48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(36, 118, 217);border-style: solid</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;background-color: rgb(255, 255, 255);border-style: solid</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;color: rgb(36, 118, 217);border-style: solid</string>
    </property>
    <property name="text">
     <string>Bank</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>58</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon.qrc">:/Icon/Icon/pay-blue-48.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>511</width>
      <height>311</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>511</width>
      <height>311</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>511</width>
      <height>311</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_10</zorder>
   <zorder>label_6</zorder>
   <zorder>buttonDeposit</zorder>
   <zorder>buttonWithdraw</zorder>
   <zorder>buttonEdit</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>buttonExit</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
